src/DETHRACE/common/car.c

```diff
@@ -5035,17 +4932,13 @@ int CollideCamera2(br_vector3* car_pos, br_vector3* cam_pos, br_vector3* old_cam
                         tv.v[2] = tv2.v[2] + tv.v[2];
                     }
                 }
+                BrVector3Add(cam_pos, car_pos, &tv);
             }
-            cam_pos->v[0] = car_pos->v[0] + tv.v[0];
-            cam_pos->v[1] = car_pos->v[1] + tv.v[1];
-            cam_pos->v[2] = car_pos->v[2] + tv.v[2];
```

Comment:
@dethrace-labs 
This little snippet.

Reply:
thanks for pointing it out. I agree, you have it it in the right place 👍

Reply:
Agree, you have this in the right place.